gcc/config/arc64/arc64.md

```diff
@@ -1189,11 +1189,12 @@ xorl"
 
 ;FIXME! add short variant for jump
 (define_insn "*sibcall<mode>_insn"
-  [(call
+ [(parallel [(call
```

Comment:
Could you please explain this change? Is it really necessary? See p.2, paragraph 2 here https://gcc.gnu.org/onlinedocs/gccint/Patterns.html:
If the vector has multiple elements, the RTL template is treated as a parallel expression.

Reply:
Done